articles/azure-functions/durable-functions-timers.md

```diff
@@ -32,14 +32,14 @@ When you create a timer that expires at 4:30 pm, the underlying Durable Task Fra
 
 ## Usage for delay
 
-The following example illustrates how to use durable timers for delaying execution. The example is issuing a billing notification every day for ten days.
+The following example illustrates how to use durable timers for delaying execution. The example is issuing a billing notification every day for five days.
 
```

Comment:
The warning right above states that "Durable timers cannot last longer than 7 days due to limitations in Azure Storage.". Changed the code so that it doesn't create timers that exceed 7 days.

Reply:
The code only shows creating 1-day timers, so this should not be in violation of the warning, right?

Reply:
I think I read `TimeSpan.FromDays(1)` as `TimeSpan.FromDays(**i**)`